cypress/plugins/index.js

```diff
@@ -0,0 +1,22 @@
+/// <reference types="cypress" />
```

Comment:
Didn't get the point of this file

Reply:
Yes. This file comes by default with cypress installation. We may require it in future when we start writing more deeper test cases.